filebeat/docs/multiline.asciidoc

```diff
@@ -0,0 +1,138 @@
+[[multiline-examples]]
+=== Managing Multiline Messages
+
+You can specify `multiline` settings in the +{beatname_lc}.yml+ file to control how Filebeat deals with messages that
+span multiple lines. At a minimum, you need to configure:
+
+* the `pattern` option, which specifies a regular expression. Depending on how you configure other multiline options, 
+lines that match the specified regular expression are considered either continuations of a previous line or the start of a new multiline event. You can set the `negate` option to negate the pattern.
+
+* the `match` option, which specifies how Filebeat combines matching lines into an event. You can specify `before` or `after`.
+
+See the full documentation for <<multiline>> to learn more about these options. Also read <<yaml-tips>> and 
+<<regexp-support>> to avoid common mistakes.
+
+==== Testing Your Regexp Pattern for Multiline
```

Comment:
We should definitively link to this in https://www.elastic.co/guide/en/beats/filebeat/master/regexp-support.html